README.md

```diff
@@ -4,6 +4,25 @@
   <img width="300" src="./images/crunchy_logo.png?raw=true"/>
 </p>
 
+## Deprecation Notice
+
+The Crunchy Containers project is no longer actively maintained, but is made available to the extent it is
+helpful as a reference to the community.  Importantly, the Crunchy Containers project is not intended for
+use in connection with the latest PGO project development.
+
+The Crunchy Containers project was released prior to the broad popularity of Kubernetes and the
+[Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).  While the Crunchy
+Containers project evolved over the years for use in conjunction with Kubernetes and Operators, with the
+evolution and the architectural changes in PGO, the approaches taken within the Crunchy Containers project
+were no longer the right solutions.
+
+For active Crunchy Data Postgres on Kubernetes projects, please see the
+[PGO repository](https://github.com/CrunchyData/postgres-operator) and the
+[PGO Examples repository](https://github.com/CrunchyData/postgres-operator-examples).  To download the latest Crunchy
+Postgres for Kubernetes containers, please see the [Crunchy Developer Portal](https://www.crunchydata.com/developers). 
+
+For community support with Postgres on Kubernetes, please [join the PGO project community Discord](https://discord.com/invite/a7vWKG8Ec9). 
```

Comment:
🤔 I'm inundated with our product name and phrasing, so "Postgres on Kubernetes" stuck out. 🤷🏻 👍🏻